test/watcher_test.rb

```diff
@@ -37,11 +37,6 @@
       message = "predictive_load: detected n+1 call on User#comments
 predictive_load: expect to prevent 1 queries
 predictive_load: would preload with: SELECT \"comments\".* FROM \"comments\"  WHERE \"comments\".\"public\" = 't' AND \"comments\".\"user_id\" IN (1, 2)
-predictive_load: 0|0|0|SCAN TABLE comments
-
-predictive_load: 0|0|0|EXECUTE LIST SUBQUERY 1
-
-predictive_load: would have prevented all 1 queries
```

Comment:
Do we not log these anymore? Or is it not preventing these queries?

Reply:
The query is still being prevented, it's just that the logging has changed since travis is now using a much newer version of sqlite so the way the explain query plan is printed is different (sqlite explicitly advises against relying on this print out https://www.sqlite.org/eqp.html).  To confirm that nothing was broken, we did go back and test this with earlier versions of sqlite and it's all good w/ 3.8.2, which is what was being used at the time.

That being said, I'm going to go ahead and remove both watcher and watcher_test since we're only supporting rails >= 4.2.0 and watcher is not supported for rails 4.1+